pkg/cmd/cli/cmd/importimage.go

```diff
@@ -52,6 +52,7 @@ func NewCmdImportImage(fullName string, f *clientcmd.Factory, out, errout io.Wri
 	cmd.Flags().StringVar(&opts.From, "from", "", "A Docker image repository to import images from")
 	cmd.Flags().BoolVar(&opts.Confirm, "confirm", false, "If true, allow the image stream import location to be set or changed")
 	cmd.Flags().BoolVar(&opts.All, "all", false, "If true, import all tags from the provided source on creation or if --from is specified")
+	cmd.Flags().StringVar(&opts.ReferencePolicy, "reference-policy", sourceReferencePolicy, "Allow to request pullthrough for external image when set to 'local'. Defaults to 'source'.")
```

Comment:
They are in [pkg/cmd/cli/cmd/tag.go](https://github.com/openshift/origin/blob/9e4e2e788bd6d42f2258efe218c71af1c62a64b2/pkg/cmd/cli/cmd/tag.go#L79-L80).